include/copc-lib/io/reader.hpp

```diff
@@ -53,5 +54,26 @@ class Reader : public BaseIO
 
     std::vector<Entry> ReadPage(std::shared_ptr<Internal::PageInternal> page) override;
 };
+
+class FileReader : private Reader
+{
+  public:
+    FileReader(const std::string &file_path)
+    {
+        auto fstream = new std::fstream;
```

Comment:
```suggestion
        this->in_stream_ = new std::fstream();
        this->in_stream_->open(file_path.c_str(), std::ios::in | std::ios::binary);
```

Reply:
For some reason I can't call open if I do that.